zone/tradeskills.cpp

```diff
@@ -622,9 +641,9 @@ void Object::HandleAutoCombine(Client* user, const RecipeAutoCombine_Struct* rac
 	}
 
     //pull the list of components
-	std::string query = StringFormat("SELECT tre.item_id, tre.componentcount "
+	std::string query = fmt::format("SELECT tre.item_id, tre.componentcount "
```

Comment:
`const auto` maybe? Do we need the alias in the query if you're only referencing a single table and can use the column names without the need for an alias?

Reply:
cleaned up.